README.md

```diff
@@ -16,45 +28,75 @@ video:
 
 - Dynamic Instrumentation (No Code Change Required)
 - Robust Sampling
-- Fast Debug Symbol Parsing
-- Context Switch Tracking
-- Disassembly View
+- Wine/Proton Mixed-Callstack Profiling
+- Thread Scheduling and Dependency Tracing
+- Memory Tracing
+- GPU Driver Tracepoints (AMD Only)
+- Vulkan Debug Label and Command Buffer Tracing (AMD Only)
+- Manual Instrumentation
+- Source Code and Disassembly View
 - Remote Profiling
-- User Sessions
+- Fast Debug Symbol Parsing
```

Comment:
Honestly I would remove "Fast", it doesn't mean much.

Reply:
Agree, too. Done.